src/toxcore/dht/server/mod.rs

```diff
@@ -132,7 +133,7 @@ pub struct Server {
     lan_discovery_enabled: bool,
     /// If IPv6 mode is enabled `Server` will send packets to IPv6 addresses. If
     /// it's disabled such packets will be dropped.
-    is_ipv6_mode: bool,
+    pub is_ipv6_enabled: bool,
```

Comment:
why pub?

Reply:
https://github.com/tox-rs/tox/blob/24cfbb1821fa9e71e4832544e3011f14c6fe8f73/src/toxcore/dht/daemon_state.rs#L40

Reply:
well you may add getter for this field.